include/jemalloc/internal/mutex_prof.h

```diff
@@ -36,21 +36,26 @@ typedef enum {
 } mutex_prof_arena_ind_t;
 
 #define MUTEX_PROF_UINT64_COUNTERS					\
-    OP(num_ops, uint64_t, "n_lock_ops")					\
-    OP(num_wait, uint64_t, "n_waiting")					\
-    OP(num_spin_acq, uint64_t, "n_spin_acq")				\
-    OP(num_owner_switch, uint64_t, "n_owner_switch")			\
-    OP(total_wait_time, uint64_t, "total_wait_ns")			\
-    OP(max_wait_time, uint64_t, "max_wait_ns")
+    OP(num_ops, uint64_t, "n_lock_ops", false, num_ops)					\
+    OP(num_ops_ps, uint64_t, "n_lock_ops_ps", true, num_ops)				\
```

Comment:
We don't really need the per second values to be real counters. They are derived so we only need to compute on demand, i.e. when output. It can be done similar to `util` value https://github.com/jemalloc/jemalloc/blob/dev/src/stats.c#L364, which is also derived and only output to the human format only.

Reply:
They are not real. The 4th argument is "synthetic" that is true for rates and false for originals. The synthetic counters are never ctl-ed but computed on demand based on where the "real" counter is. And the name of the "real" counter is provided with the 5th argument of the macro.

Having the synthetics inserted inside the sequence of OP() makes us able to have synthetics next to the real countners and not to keep them separate.

Reply:
I see. To keep the terms consistent, let's use `derived` for the synthetic ones. Also we would need some comments to mention that the derived ones are used for rates currently.